simulator/container.go

```diff
@@ -268,6 +286,42 @@ func (c *container) start(ctx *Context, vm *VirtualMachine) {
 	if err = c.inspect(vm); err != nil {
 		log.Printf("%s inspect %s: %s", vm.Name, c.id, err)
 	}
+
+	// Keep updating the VM properties by inspecting the container until
+	// an inspect call fails.
+	inspectInterval := time.Duration(5 * time.Second)
+	if d, err := time.ParseDuration(os.Getenv("VCSIM_INSPECT_INTERVAL")); err == nil {
+		inspectInterval = d
+	}
+
+	go func() {
+		if inspectInterval == 0 {
+			return
+		}
+		for {
+			time.Sleep(inspectInterval)
```

Comment:
Just realized, if the vm was destroyed from the API side (rather than docker rm), `c.inspect` will return `nil` and I think this go routine will never exit.  If that's the case, just adding this here should do the trick:
```go
	if c.id == "" {
		return
	}
```
https://github.com/vmware/govmomi/blob/dfe03eb4b4f6e0c825a81b70caf9fab7181649e4/simulator/container.go#L64-L66